src/Editor/Resources_BACKUP_7056.cpp

```diff
@@ -0,0 +1,261 @@
+#include "Resources.hpp"
```

Comment:
This whole file should be removed. We do not want conflict and backup files in the master thread.